ark/type/methods/base.ts

```diff
@@ -140,8 +225,20 @@ interface Type<out t = unknown, $ = {}>
 			| PredicateCast<this["inferIn"], narrowed>
 	): instantiateType<r, $>
 
+	/**
+	 * Construct an array of this type.
+	 * @example const array = type("string").array() // Type<string[]>
+	 * @example const tupleForm = type(["string", "[]"])
+	 * @example const argsForm = type("string", "[]")
+	 */
 	array(): ArrayType<t[], $>
 
+	/**
+	 * Morph this type with a chain of morphs.
```

Comment:
This shouldn't show the different syntax kinds. Just summarize how the chaining works, and explain that types can be morphed through other types which will apply validation and morphs from those types.